### ISSUE TITLE
problems on Windows 10 / Java 10

### ISSUE THREAD

**Issue by smack42** [smack42/DriftingDroids#11]
It was reported today by e-mail that DriftingDroids doesn't run on Windows 10 systems using Java 10.

**Comment by smack42** [comment]
The application starter inside of "start.jar" doesn't work in Java 10.
Error message is:
```
Exception in thread "main" java.lang.ClassCastException: java.base/jdk.internal.loader.ClassLoaders$AppClassLoader cannot be cast to java.base/java.net.URLClassLoader
        at org.appstart.Starter.main(Starter.java:75)
```

Apparently this is caused by a change that was part of Java 9:
http://www.oracle.com/technetwork/java/javase/9-relnote-issues-3704069.html
> The application class loader is no longer an instance of java.net.URLClassLoader (an implementation detail that was never specified in previous releases). Code that assumes that ClassLoader::getSytemClassLoader returns a URLClassLoader object will need to be updated. Note that Java SE and the JDK do not provide an API for applications or libraries to dynamically augment the class path at run-time.

**Comment by smack42** [comment]
btw. the application starter we're using is this one:
https://code.google.com/archive/p/appstart/
https://github.com/neophob/appstart  (a fork of the one above)

**Comment by smack42** [comment]
Updated appstart fixes this issue

**Comment by smack42** [comment]
Published preview version for testing of the fix for this issue.
https://github.com/smack42/DriftingDroids/releases/download/1.3.6-dev_2018-04-21/DriftingDroids_1.3.6-dev_2018-04-21.zip

**Comment by smack42** [comment]
Successfully tested using:
- OpenJDK 8 on Linux
- Oracle Java 8 on Windows 7
- Oracle Java 10 on Windows 7
- Oracle Java 8 on Windows 10
- Oracle Java 10 on Windows 10

**Comment by smack42** [comment]
solved by release 1.3.6
https://github.com/smack42/DriftingDroids/releases
